uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget-absence-types-amount-taken.component.js

```diff
@@ -60,8 +60,9 @@ define([
      * @return {Boolean}
      */
     function areBindingsReady () {
-      return vm.absenceTypes && vm.contactId && vm.absencePeriod &&
-        vm.leaveRequestStatuses && vm.leaveRequestStatuses.length;
+      return vm.absenceTypes && vm.absenceTypes.length && vm.contactId &&
```

Comment:
`vm.absenceTypes && vm.absenceTypes.length`

why do you need to check for both?

Reply:
`absenceTypes` is passed as a binding and might be undefined at first, and then it might be an empty array, and finally it will have a list of absence types, which is needed for displaying the absence types taken.